### PR TITLE
feat: upload Dockerfiles as part of the release process

### LINKED ISSUE #733: Upload Dockerfiles as part of the release
The Dockerfiles should be uploaded as part of the release assets so that they can be easily accessed for download.

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says

Closes #733

### CODE REVIEW

**Comment by RobPasMue** [comment]
@AlejandroFernandezLuces - please review

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -540,6 +553,26 @@ jobs:
...
+          retention-days: 7
+
+      - name: Upload Linux Dockerfile
+        uses: actions/upload-artifact@v3
+        with:
+          name: linux-dockerfile.zip
+          path: docker/Dockerfile.linux
+          retention-days: 7
+
       - name: Stop the Geometry service
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:576]:
```suggestion
      - name: Stop the Geometry service
```

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -564,7 +590,7 @@ jobs:
         uses: ansys/actions/release-github@v4
         with:
           library-name: ${{ env.PACKAGE_NAME }}
-          additional-artifacts: windows-binaries.zip linux-binaries.zip
+          additional-artifacts: windows-binaries.zip linux-binaries.zip linux-dockerfile.zip windows-dockerfile.zip
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:593]:
```suggestion
          additional-artifacts: windows-binaries.zip windows-dockerfile.zip linux-binaries.zip linux-dockerfile.zip
```

**Review by AlejandroFernandezLuces — APPROVED** [review]
LGTM!

**Pull request merged**